camel/messages/base.py

```diff
@@ -198,7 +212,37 @@ def to_openai_user_message(self) -> OpenAIUserMessage:
         Returns:
             OpenAIUserMessage: The converted :obj:`OpenAIUserMessage` object.
         """
-        return {"role": "user", "content": self.content}
+        if self.image is None:
+            return {"role": "user", "content": self.content}
+        else:
+            if self.image.format is None:
+                raise ValueError(f"Only support image with specified format, "
+                                 f"including {list(OpenAIImageType)}")
```

Comment:
The error message is unrelated to the if condition. One checks for None, while the message indicates that the image format is unexpected.